auth-web/tests/unit/views/HomeView.spec.ts

```diff
@@ -17,11 +17,16 @@ const vuetify = new Vuetify({})
 // Prevent the warning "[Vuetify] Unable to locate target [data-app]"
 document.body.setAttribute('data-app', 'true')
 
+const mockSession = {
+  'NRO_URL': 'Mock Url'
```

Comment:
You set this; is it possible to have a test that clicking the button attempts to redirect to it (same as we do in some Filings UI unit tests)?

Reply:
Absolutely, for the sake of unblocking the other team, are you ok if i defer expanding the unit tests to a future commit?

Reply:
Yes, absolutely.